pkg/envoy/envoy.go

```diff
@@ -94,6 +94,21 @@ type Logger interface {
 	Log(entry *HttpLogEntry)
 }
 
+// LogEnvoyVersion logs envoy binary version
+func LogEnvoyVersion() {
+	out, err := exec.Command("cilium-envoy", "--version").Output()
+	if err != nil {
+		log.WithError(err).Fatal(`Envoy binary "cilium-envoy" cannot be executed`)
+	}
+	start := 0
+	end := len(out)
+	for ; start < end && out[start] == '\n'; start++ {
+	}
+	for ; start < end && out[end-1] == '\n'; end-- {
+	}
```

Comment:
Could you add comment explaining what we're trying to do here? It's not apparent that the version output has the leading and trailing newlines.
Alternatively, a regex to match the line, and extract the version directly, might be more resilient? In particular, if the version output ever added another line that would print oddly in the Infof below.

Reply:
Or https://golang.org/pkg/strings/#TrimSpace

Reply:
Nice, using this now!